crates/release_plz/src/args/release.rs

```diff
@@ -43,18 +44,33 @@ pub struct Release {
     /// Git token used to publish the GitHub release.
     #[clap(long, value_parser = NonEmptyStringValueParser::new())]
     pub git_token: Option<String>,
+    /// Kind of git backend
+    #[arg(long, value_enum, default_value_t = GitBackendKind::Github)]
+    backend: GitBackendKind,
 }
 
 impl TryFrom<Release> for ReleaseRequest {
     type Error = anyhow::Error;
 
     fn try_from(r: Release) -> Result<Self, Self::Error> {
+        let git_token = SecretString::from(
+            r.clone()
+                .git_token
+                .context("git_token is required for git_release")?,
```

Comment:
git_token is required only if the `git_release` flag is set.
To fix this, you can move this assignment inside `if r.git_release {`. I.e. move to line 62.